src/libasr/pass/print_arr.cpp

```diff
@@ -256,6 +256,18 @@ class PrintArrVisitor : public PassUtils::PassVisitor<PrintArrVisitor>
         return doloop;
     }
 
+    void print_args_apart_from_arrays(std::vector<ASR::expr_t*> &write_body, const ASR::FileWrite_t& x) {
+        Vec<ASR::expr_t*> body;
+        body.reserve(al, write_body.size());
+        for (size_t j=0; j<write_body.size(); j++) {
+            body.push_back(al, write_body[j]);
+        }
```

Comment:
```suggestion
        Vec<ASR::expr_t*> body;
        body.from_pointer_n(&write_body[0], write_body.size())
```
I forgot the function name, try `from_pointer_n` or `from_pointer_n_copy`